doc/build-unix.md

```diff
@@ -248,3 +248,27 @@ Additional Configure Flags
 A list of additional configure flags can be displayed with:
 
     ./configure --help
+
+
+
+Cross-compilation
+-------------------
+These steps can be performed on, for example, an Ubuntu VM. The depends system
+will also work on other Linux distributions, however the commands for
+installing the toolchain will be different.
+
+First install the toolchain:
+
+    sudo apt-get install g++-arm-linux-gnueabi
+
+To build executables for ARM:
+
+    cd depends
+    make -j10 HOST=arm-unknown-linux-gnueabihf NO_QT=1 # NO_WALLET=1 NO_UPNP=1 ...
```

Comment:
Not sure I agree with `-j10` as a default for documentation. Better leave it out entirely as is the case for existing documentation.